web/views/changeset_view.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Jingle.ChangesetView do
```

Comment:
Hrmm... not sure i understand what this is.  But I'll persevere and keep reading!

Reply:
pretty sure phoenix made this all by itself